code/gbq/configs/gbq_qr_hhs_only.py

```diff
@@ -0,0 +1,8 @@
+import copy
+from configs.base import base_config
+
+config = copy.deepcopy(base_config)
+config.model_name = 'gbq_qr_hhs_only'
+
+config.bag_frac_samples = 1
+config.sources = ['hhs']
```

Comment:
set source to use only hhs.  Also override `bag_frac_samples` to use all samples in all bags.  Necessary with only one historical season.